tests/RobotFramework/MQTT_health_check/MQTT_health_endpoints.robot

```diff
@@ -0,0 +1,53 @@
+*** Settings ***
+Library    SSHLibrary
+Suite Setup            Open Connection And Log In
+Suite Teardown         SSHLibrary.Close All Connections
+
+*** Variables ***
+${HOST}           192.168.100.110
+${USERNAME}       pi
+${PASSWORD}       thinedge
+${pid}
+*** Tasks ***
+
+Connect to Cumulocity
+    ${rc}=    Execute Command    sudo tedge connect c8y
```

Comment:
So, the assumption is that the `thin-edge` is already is installed right? I feel it's difficult to run an individual test. Better to check if `thin-edge` exists on that box or not right?